helix-view/src/editor.rs

```diff
@@ -289,6 +296,7 @@ impl Editor {
             idle_timer: Box::pin(sleep(config.idle_timeout)),
             last_motion: None,
             config,
+            auto_pairs,
```

Comment:
This is actually not allowed at this point because we move the `config` in the line above, so we can't reference a field of it later. I'll change the invocation to an `into()` though.